content/docs/concepts/assets.txt

```diff
@@ -0,0 +1,4 @@
+assets can be found here:
+
+https://docs.google.com/presentation/d/1hoialQsjCs1PPwo4TIp8fT8EGnU9SM-A/edit#slide=id.p1
```

Comment:
Can you please change the link from `edit` to `view` mode.

Reply:
Updated it. Hopefully this does the trick.